rerun_py/src/python_bridge.rs

```diff
@@ -79,7 +82,7 @@ type GarbageReceiver = crossbeam::channel::Receiver<ArrowRecordBatch>;
 /// accumulated data for real.
 //
 // NOTE: `crossbeam` rather than `std` because we need a `Send` & `Sync` receiver.
-static GARBAGE_QUEUE: Lazy<(GarbageSender, GarbageReceiver)> =
+pub(crate) static GARBAGE_QUEUE: Lazy<(GarbageSender, GarbageReceiver)> =
```

Comment:
nit: here and below, why not just `pub` given that `lib.rs` doesn't do blanket re-export

Reply:
i don't need to re-export them even, for the original implementation done at the kitchen table I had to, but not anymore 🕺